test/Microsoft.ML.OnnxTransformerTest/DnnImageFeaturizerTest.cs

```diff
@@ -173,5 +174,61 @@ public void TestOldSavingAndLoading()
                 }
             }
         }
+
+        internal sealed class ModelInput
+        {
+            [ColumnName("ImagePath"), LoadColumn(0)]
+            public string ImagePath { get; set; }
+
+            [ColumnName("Label"), LoadColumn(1)]
+            public string Label { get; set; }
+        }
+
+        internal sealed class ModelOutput
+        {
+            // ColumnName attribute is used to change the column name from
+            // its default value, which is the name of the field.
+            [ColumnName("PredictedLabel")]
+            public String Prediction { get; set; }
+            public float[] Score { get; set; }
+        }
+
+        [Fact]
+        public void TestLoadFromDiskAndPredictionEngine()
+        {
```

Comment:
I wasn't sure if it was okay to add this test in here, in the DnnImageFeaturizerTest.

Originally I intended to add this as a Functional Test, but I believe I couldn't do that because I couldn't access ResNet18 unless, I believe, I changed the dependencies there. So I added the test in here instead.

Also, in order to make this test to run in here I had to modify the project `OnnxTransformerTest.csproj ` to include `CpuMathNative`, because if I didn't do that, then the test threw an error.

In short, I wasn't sure where to put this test; if this isn't the place, please tell me so I can make the appropriate changes. Thanks. @codemzs

Reply:
Also, I wonder if it should use [OnnxFact] instead of [Fact]??

Reply:
Also, I don't know if to include CpuMathNative into the OnnxTransformerTest.csproj the only thing I had to do was to add a line of code (which I did). I wonder if I had to do something else.

Reply:
Before changing [Fact] to [OnnxFact] some tests were failing, but now all of them pass. So I guess it is correct to make that change... but I still wonder if this is the right place for this test.

Reply:
So I discussed this offline with @codemzs and all of these is fine. So I'll mark this as resolved.